src/FsLex/fslexast.fs

```diff
@@ -18,6 +15,7 @@ let Eof : Alphabet = 0xFFFFFFFEu
 let Epsilon : Alphabet = 0xFFFFFFFFu
 
 let unicode = ref false
+let caseInsensitive = ref false
```

Comment:
Both of these are now better as "let mutabl;e caseInsensitive = false"

Reply:
Refactored to mutable.